core/src/test/java/io/undertow/websockets/client/version13/WebSocketClient13TestCase.java

```diff
@@ -60,6 +61,7 @@
  */
 @RunWith(DefaultServer.class)
 @HttpOneOnly
+@Ignore
```

Comment:
I am assuming you are using this for testing purposes temporarily, for that reason I'm ignoring this for now

Reply:
Oh my, completely forgot about this, yeah. This test sometimes "randomly fails consistently".

Reply:
I think this one was fixed by a PR sent by Ricardo.

Reply:
(already merged, the one that solved a TCK issue as well)